TriblerGUI/test/widgets/trustpage/js/test_drawing.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This file tests the drawing.js file with unit tests
+ */
+assert = require("assert");
+drawing = require("../../../../widgets/trustpage/js/drawing.js");
+
+describe("drawing.js", function () {
+    describe("getStrokeWidth", function () {
```

Comment:
This may look trivial from your perspective as you wrote this code, but to me it's not. I hope you will reconsider.